src/translators/csharp.rb

```diff
@@ -105,7 +105,7 @@ def docs_signatures_for(function)
           end
 
           result.unshift "public #{method_data[:static]}#{method_data[:return_type]} #{method_data[:class_name]}.#{property_name.to_pascal_case()} { #{text} }"
-        else
+        elsif method_data[:method_name]
```

Comment:
Checking whether method_name exists before referencing it makes sense to avoid errors. However, it does mean that method_name may be null.